src/test/java/com/google/cloud/genomics/dataflow/pipelines/VerifyBamIdTest.java

```diff
@@ -15,6 +15,16 @@
  */
 package com.google.cloud.genomics.dataflow.pipelines;
 
+import static org.junit.Assert.assertEquals;
+
+import java.util.Map;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.beust.jcommander.internal.Lists;
```

Comment:
Hi Nicole,

Why do you need to import jcommander?  The code is basically the following:

```Java
(List <String>) new ArrayList<String>(Arrays.asList( (String[]) stringValue)))
```

You can even write a function for the above for the different instances used: `Lists.newArrayList("G")`, `Lists.newArrayList("T")`, and `Lists.newArrayList("C")`.

Hope it simplifies,
~p